app/controllers/users_controller.rb

```diff
@@ -0,0 +1,34 @@
+class UsersController < ApplicationController
+  before_action :load_user, only: :show
+
+  def show
+    @user = User.find(params[:id])
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new user_params
+
+    if @user.save
+      flash[:success] = t "user.new.welcome"
+      redirect_to @user
+    else
+      render :new
+    end
+  end
+
+  private
+
+    def user_params
```

Comment:
This function should straight with private

Reply:
Ok done